pandas/tseries/base.py

```diff
@@ -26,7 +25,7 @@ def strftime(self, date_format):
         Return an array of formatted strings specified by date_format, which
         supports the same string format as the python standard library. Details
         of the string format can be found in the `python string format doc
-        <https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior>`__
+        https://docs.python.org/2/library/datetime.html.
```

Comment:
This will give an error with Sphinx, as you left in the opening `` ` ``. 
But I personally also find the original link better.